apps/static-site/translations/en.json

```diff
@@ -22,5 +24,11 @@
   "Contact and accessibility": "Contact and accessibility",
   "Article": "Article",
   "Table of contents": "Table of contents",
-  "All items": "All items"
+  "homeTitle": "TU Delft Library's Special Collections portal",
+  "homeDesc": "Home description en",
+  "aboutDesc": "About description en",
+  "searchDesc": "Search description en",
+  "exhibitionsDesc": "Exhibitions listing description en",
+  "collectionsDesc": "Collections listing description en",
+  "publicationsDesc": "Publications listing description en"
```

Comment:
@sammeltassen Would you be able to provide some short descriptions that will appear in the social media cards. Thanks!